InternshipBE/BL/Interfaces/IInternshipStackService.cs

```diff
@@ -6,10 +6,12 @@ namespace BL.Interfaces
 {
     public interface IInternshipStackService
     {
+        Task<List<GetInternshipStackDTO>> GetInternshipStacksAsync();
+
         Task<List<GetInternshipStackDTO>> GetInternshipStacksByInternshipIdAsync(int internshipId);
 
-        Task<FullInternshipStackDTO> CreateInternshipStackAsync(CreateInternshipStackDTO internshipStackDto);
+        Task<GetInternshipStackDTO> CreateInternshipStackAsync(InternshipStackDTO internshipStackDto);
```

Comment:
out GetInternshipStackDTO and in InternshipStackDTO ?

Reply:
We get dto without id, create entity, and return dto with autogenerated id